easybuild/easyconfigs/i/ieeg-cli/ieeg-cli-1.14.56.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'Tarball'
+
+name = 'ieeg-cli'
+version = '1.14.56'
+
+homepage = 'https://www.ieeg.org/'
+description = """IEEG.ORG is a collaborative initiative funded by the National Institutes 
+of Neurological Disorders and Stroke. This initiative seeks to advance research towards 
+the understanding of epilepsy by providing a platform for sharing data, 
+tools and expertise between researchers."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['https://bitbucket.org/ieeg/ieeg/downloads']
+sources = ['%(name)s-%(version)s-dist.zip']
+checksums = ['fbec0bc36615ffde121e4ca11e04cbc5391003ee557f1596d84c1f53ba9530e8']
+
+dependencies = [('Java', '11.0.2')]
```

Comment:
@murfreesboro Why not use the `Java/11` wrapper instead, no need to depend on a specific `Java` version?

fixed in #8808 8811

Reply:
oops, missed that one..